apps/web/src/lib/schema/profile.ts

```diff
@@ -9,8 +9,15 @@ export const passwordSchema = z.object({
 });
 
 export const mailSettingsSchema = z.object({
-    daily: z.boolean().default(false),
-    weekly: z.boolean().default(false),
+    receiveMails: z.boolean().default(true),
+    interval: z.coerce.number().int()
+        .positive({message: "Bitte geben Sie ein gültiges Intervall an."})
```

Comment:
Eine genaue Differenzierung der Fehlermeldungen fände ich hier besser, Also in dieser Zeile sowas wie "Bitte geben Sie einen positiven Wert für das Intervall an" und dann in der nächsten "Bitte geben Sie einen Wert kleiner oder gleich 7 für das Intervall an".
Für time dann auch